charts/dbildungscloud-iam-keycloak/values.yaml

```diff
@@ -0,0 +1,4 @@
+replicaCount: 1
+keycloakImage: "ghcr.io/dbildungsplattform/dbildungs-iam:sha-15679d4"  # for test purpose i set this image name
```

Comment:
regarding the default values i am not sure if and how they get set here - i think this depends on the pipeline/structure